source/transfer_list.rst

```diff
@@ -951,6 +951,48 @@ software running in Secure, Non-Secure, or Realm modes.
      - hdr_size
      - Holds a single `entry_point_info` structure.
 
+**FF-A SP binary (XFERLIST_FFA_SP_BINARY)**
+
+This entry holds a reference to an FF-A secure partition's binary.
+
+Hafnium SPMC supports the SP's artefacts to be packaged as a Transfer List
+generated at build time (with TF-A's tlc tool). The respective Transfer List
+contents: SP binary/image, the FF-A manifest, and other optionl resoures
+(e.g. HOB).
+
+Under the the described context, this TE type is for an SPMC implementation
+to identify which entry relates to the SP's binary, such that it can
+bootstrap/hand-over execution.
```

Comment:
perhaps could be a bit more descriptive, e.g. "so that it can install the binary and hand-over execution to it".